go/teams/implicit_test.go

```diff
@@ -71,3 +70,34 @@ func TestLookupImplicitTeams(t *testing.T) {
 	_, _, err = LookupOrCreateImplicitTeam(context.TODO(), tc.G, "dksjdskjs/sxs?", true)
 	require.Error(t, err)
 }
+
+// Test an implicit team where one user does not yet have a PUK.
+func TestImplicitPukless(t *testing.T) {
+	fus, tcs, cleanup := setupNTestsWithPukless(t, 2, 1)
+	defer cleanup()
+
+	displayName := "" + fus[0].Username + "," + fus[1].Username
+	t.Logf("U0 creates an implicit team: %v", displayName)
+	teamID, _, err := LookupOrCreateImplicitTeam(context.Background(), tcs[0].G, displayName, false /*isPublic*/)
+	require.NoError(t, err)
+
+	// TODO enable this after fixing lookup
+	// teamID2, _, err := LookupOrCreateImplicitTeam(context.Background(), tcs[0].G, displayName, false /*isPublic*/)
+	// require.NoError(t, err)
+	// require.Equal(t, teamID, teamID2)
```

Comment:
There's still a problem where Lookup doesn't work for this team. I'll look into that and possibly put a fix here, but it shouldn't be necessary to merge this PR.